### PR TITLE
Jt/pm display data

### PR DESCRIPTION
**Pull request by JasperTan-03**

### CODE REVIEW

File: frontend/src/Components/Manager/Manager.jsx [hunk 1]
```diff
@@ -6,10 +6,44 @@ import './Manager.css';
...
   // Sample project data, replace with your project data from MongoDB
   projects = [
-    { id: 1, name: 'Project 1' },
-    { id: 2, name: 'Project 2' },
-    { id: 3, name: 'Project 3' },
-    // Add more projects here
+    {
+      projectId: 1,
+      projectName: 'Sample Project 1',
+      authorizedUsers: ['Dumb Troy', 'Happy Haakon', 'BOOP'], // Example authorized users
```
Comment (Bushvacka) [inline frontend/src/Components/Manager/Manager.jsx:12]: Sorry could we change these please?

**Review by HaakonMongstad — APPROVED** [review]
Wow jasper. This code is simply stellar. You never fail to amaze me!

**Review by Bushvacka — CHANGES_REQUESTED** [review]
Jaspy tan

File: frontend/src/Components/Manager/Manager.jsx (hunk 1 above)
Comment (JasperTan-03) [reply frontend/src/Components/Manager/Manager.jsx:12]: Hi Troy,
No

**Pull request merged**